src/python/ImpactX.cpp

```diff
@@ -272,22 +275,22 @@ void init_ImpactX (py::module& m)
              "distribution's extent and then redistribute particles in according\n"
              "AMReX grid boxes."
         )
+
         .def("evolve", &ImpactX::evolve,
              "Run the main simulation loop for a number of steps."
         )
+        // TODO: step
+        .def("resize_mesh", &ImpactX::ResizeMesh,
+             "Resize the mesh, based on the :py:attr:`~domain` extent."
+        )
+
         .def("particle_container",
              [](ImpactX & ix) -> ImpactXParticleContainer & {
                 return *ix.m_particle_container;
              },
              py::return_value_policy::reference_internal,
              "Access the beam particle container."
         )
-        .def("reduced_beam_characteristics",
```

Comment:
@n01r @RTSandberg I moved  this now over to the ImpactX particle container after all, where it is more natural with the other methods I added.